setup.html

```diff
@@ -5,18 +5,31 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Setup</title>
+    <title>Mixtape Setup</title>
+
+    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+    <link rel="stylesheet" href="assets/css/style.css">
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker|Roboto+Mono:400,700"" rel=" stylesheet">
 </head>
 
 <body>
-    <section id="playlist-selection">
-        <h1>Select a playlist</h1>
-        <select id="playlist">
-            <option value="">Select from your playlist</option>
-        </select>
-        <button id="setup">Setup</button>
-    </section>
-    <section id="error"></section>
+    <header class="masthead d-flex">
+        <div class="container text-center my-auto">
+            <img src="assets/images/logo_no-shadow.png" class="img-fluid mx-auto d-block" id="mixtape">
+
+            <section id="playlist-selection">
+                <div class="btn-group">
+                    <select class="form-control" id="playlist">
+                        <option value="">SELECT FROM YOUR PLAYLISTS</option>
+                    </select>
+                    <a class="btn btn" id="setup">LOAD</a>
```

Comment:
Same here about the duplicated `btn` class.

Reply:
checked